kahypar/partition/coarsening/heavy_edge_rater.h

```diff
@@ -128,6 +133,24 @@ class HeavyEdgeRater {
   HypernodeWeight thresholdNodeWeight() const {
     return _config.coarsening.max_allowed_node_weight;
   }
+  
+  void resetCommunities() {
+      _comm.assign(_comm.size(),0);
+  }
```

Comment:
Where does this method come from? Is this still needed somewhere or can we delete it?